app/src/main/java/com/amaze/filemanager/fragments/MainFragment.java

```diff
@@ -688,6 +688,7 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                     } else {
                         adapter.toggleChecked(true, CURRENT_PATH);
                     }
+                    item.setTitle(item.getTitle().equals(getResources().getString(R.string.selectall)) ? R.string.deselect_all : R.string.selectall);
```

Comment:
`item.getTitle().equals(getResources().getString(R.string.selectall)` in O(n), can't you use a variable?

Reply:
See 709ed2f.